docs/_docs/internals/syntax.md

```diff
@@ -254,7 +254,7 @@ InfixExpr         ::=  PrefixExpr
                     |  InfixExpr MatchClause
 MatchClause       ::=  ‘match’ <<< CaseClauses >>>                              Match(expr, cases)
 PrefixExpr        ::=  [PrefixOperator] SimpleExpr                              PrefixOp(expr, op)
-PrefixOperator    ::=  ‘-’ | ‘+’ | ‘~’ | ‘!’
+PrefixOperator    ::=  ‘-’ | ‘+’ | ‘~’ | ‘!’                                    -- unless backquoted
```

Comment:
The comment is not necessary, since the lexemes are spelled out. The spec text does say, "one of the identifiers...", so it must add, "which must not be enclosed in backquotes." (Where backquote is allowed, it must be shown in a production.)

Reply:
Which spec are you referring to?

Reply:
And do you think I should remove the comment? I think it's kind of ambiguous without it.

Reply:
I mean the spec under scala 2 https://scala-lang.org/files/archive/spec/2.13/06-expressions.html#prefix-operations

Personally, I find it more precise as it is. If it doesn't say you can use
```
'`' '~' '`'
```
then you can't, modulo bugs. Backquotes are not some metacharacter with respect to syntax. Or they are not onions, as in, "Would you like onions with that?"

However, there are two groups of consumers for the two syntax pages, so maybe they would say the comment is clarifying. I don't know which group, but hopefully they will voice their preference.